Data validation new layout

breadsp opened the pull request:
Finished Major rewrite of the code
Everything works locally and in leidos

still needs to be thoroughly tested

breadsp committed:
reworked code for pandas tables

Rewrote a lot of code to optimize pandas table logic
taking advantage of built in functions such as isdigit
using apply(lambda x:x) logic versus for loops

breadsp committed:
Add files via upload

breadsp committed:
all code has been updated

Rewrote a lot of the function calls.  Every file has now been updated
Fixed bugs in Test mode.  Everything is working is leidos account

breadsp committed:
fixed codacy bugs

n2iw commented:
Looking good! Only a couple suggestions:

- all files: there are some condition blocks with "if"  and "elif" but don't have an "else", please double check if an "else" should be added to catch any unexpected errors.
- nci-seronet-data-validator.py, please double check if an inner "try, except" block pair (inside loop over files, similar to the one you added to file-validator) is needed to deal with unexpected exceptions, so other files can be processed?

breadsp committed:
applied Ming sugestions

put code in a try/except loop incase error occurs during processing
Added in else statements to each validation file to catch unexpected header names (should not happen due to file-validator check)
Added in SQL commands to update jobs tables with FAILURE/ERROR/SUCCESS options based on errors found in validation

breadsp committed:
fixed codacy sugestions

n2iw commented:
There are still some if, elif blocks without else block. I assume those are by design, and will merge the PR.
